arrow/arrow.py

```diff
@@ -517,6 +518,8 @@ def span(
         :param exact: (optional) whether to have the start of the timespan begin exactly
             at the time specified by ``start`` and the end of the timespan truncated
             so as not to extend beyond ``end``.
+        :param week_start: (optional) only used in combination with frame == "week". Follows isoweekday() where Monday
```

Comment:
I think we should change the comment a bit to say "only used in combination with the week timeframe. Follows..."

Reply:
Updated comment.